docs/production/install.md

```diff
@@ -119,7 +119,7 @@ server.
 ```eval_rst
 .. _installer-details:
 ```
-## Details: What the installer does
+## Details: what the installer does
```

Comment:
I'm not sure about the role of "what" here structurally and thus whether it should be captilized?